lessons/introR/main.Rmd

```diff
@@ -0,0 +1,291 @@
+---
```

Comment:
I do link to the Rmd file from within the document, on line 279. Not sure if it works in the website... but so far the lessons are not directly accessible via the website anyway (unless you type out the direct url in the address bar)